templates/ts-playwright-test-runner/.actor/actor.json

```diff
@@ -8,8 +8,10 @@
 	"meta": {
         "templateId": "ts-playwright-test-runner"
     },
-	"environmentVariables": {},
 	"storages": {
-        "dataset": "./OUTPUT_SCHEMA.json"
-    }
+        "dataset": "./output_schema.json"
+    },
+	"input": "./input_schema.json",
+    "readme": "../README.md",
```

Comment:
you can drop this line, we had a short discussion about this today and I've removed it from all the other templates too